### PR TITLE
UserMenu module

### PR DESCRIPTION
**Pull request by agiannastasio**
I added a new component for the User Menu, replicating the Logo component I found in the style guide. I'd like to discuss if this is the right approach when adding new components.

### CODE REVIEW

File: src/components/UserMenu/index.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import styles from './UserMenu.css'
+
+/**
+ * Unleash User Menu component
+ */
+ var UserMenu = React.createClass({
```
Comment (szymonmichalak) [inline src/components/UserMenu/index.js]:
```js
import React, { Component } from 'react'
...
class UserMenu extends Component {
  constructor(props) {
    super(props)
    this.state = { isChecked: false }
  }
  ...
}
```

File: src/components/UserMenu/index.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    getInitialState: function() {
+        return { isChecked: false };
+    },
```
Comment (szymonmichalak) [inline src/components/UserMenu/index.js]: Just use `constructor` as mentioned in my comment above.

File: src/components/UserMenu/index.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+							<li className={styles.UserMenuItem, styles.UserMenuItemEdit}><a href={'#'} className={styles.UserMenuItemLink}>Edit Profile</a></li>
+							<li className={styles.UserMenuItem, styles.UserMenuItemSignOut}><a href={'#'} className={styles.UserMenuItemLink}>Sign Out</a></li>
+						</ul>
+					</nav>
+			  </div>
+      );
+    }
+});
+
+module.exports = UserMenu;
```
Comment (szymonmichalak) [inline src/components/UserMenu/index.js]: `export default UserMenu`

File: src/components/UserMenu/index.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+							<li className={styles.UserMenuItem, styles.UserMenuItemName}>Nikita</li>
+							<li className={styles.UserMenuItem, styles.UserMenuItemEmail}>nikita.ormonov@eleken.co</li>
+							<li className={styles.UserMenuItem, styles.UserMenuItemEdit}><a href={'#'} className={styles.UserMenuItemLink}>Edit Profile</a></li>
+							<li className={styles.UserMenuItem, styles.UserMenuItemSignOut}><a href={'#'} className={styles.UserMenuItemLink}>Sign Out</a></li>
+						</ul>
+					</nav>
+			  </div>
+      );
+    }
+});
```
Comment (szymonmichalak) [inline src/components/UserMenu/index.js]: Please fix indent + no semicolons. Run `npm run lint` to check other issues here (`console.log`, etc.).

File: src/components/UserMenu/index.js (hunk 3 above)
Comment (agiannastasio) [inline src/components/UserMenu/index.js]: thanks for your feedback, @szymonmichalak!
btw, this PR won't be merged, as this component is already implemented on #9

**Comment by szymonmichalak** [comment]
No worries @agiannastasio. Could you please take a look at #9 locally and see if it's missing anything? Also you can compare on how @jacekelgda did that (how the components are divided, it's a bit more granular there).

**Pull request closed without merging**